app/controllers/api/v0/swagger/models/highlight.rb

```diff
@@ -42,25 +42,84 @@ module Api::V0::Swagger::Models::Highlight
     end
   end
 
+  swagger_schema :GetHighlights do
+    key :required, [:source_type]
+    property :page do
+      key :type, :integer
+      key :description, 'The page number of paginated results, one-indexed. Defaults to 1.'
```

Comment:
Ditto above on not repeating these defaults in literal strings - refer to some variable.